backend/migrations/versions/a375771e4d0f__alembic_sync_state.py

```diff
@@ -0,0 +1,185 @@
+"""_alembic_sync_state
+
+Revision ID: a375771e4d0f
+Revises: f6cd4ba7dd8d
+Create Date: 2024-02-05 11:30:56.177755
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = 'a375771e4d0f'
+down_revision = 'f6cd4ba7dd8d'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.drop_table('item_tags')
+    op.drop_table('dataset_table_profiling_job')
+    op.drop_table('redshiftcluster')
+    op.drop_table('dataset_quality_rule')
+    op.drop_table('redshiftcluster_datasettable')
+    op.drop_table('redshiftcluster_dataset')
+    op.drop_table('tag')
+    op.drop_table('group_member')
+    op.alter_column('datapipelineenvironments', 'region',
+                    existing_type=sa.VARCHAR(),
+                    nullable=True)
+    op.alter_column('dataset_bucket', 'partition',
+                    existing_type=sa.VARCHAR(),
+                    nullable=True)
+    op.drop_constraint('dataset_bucket_datasetUri_fkey', 'dataset_bucket', type_='foreignkey')
```

Comment:
In the `dataset_bucket` table instead of dropping the foreign key in RDS I think the issue is in the definition in the code. We should add the foreign key in `dataall.modules.datasets_base.db.dataset_models.DatasetBucket`